### DISCUSSION TITLE
zh/doc/recognizing-cyl/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#614] [Announcements]
# zh/doc/recognizing-cyl/

Tip：为了方便，在以下内容中部分Cylheim会用缩写Cyl替代。 开始之前 Cylheim仅支持Windows，因此请尽量确保自己电脑运行推荐Windows10以上的系统。低于Windows10的系统运行Cylheim无法确保稳定。 安装Cylheim Cyl可以通过以下两种方式安装：
微软商店（Microsoft Store） (https://...

https://cyl.yukiverse.icu/zh/doc/recognizing-cyl/

**Comment by LKWLKPI31** [comment]
我记得没有msstore可以通过命令提示符下载
